v2/sourcedb-to-spanner/src/main/java/com/google/cloud/teleport/v2/source/reader/io/jdbc/dialectadapter/mysql/MysqlDialectAdapter.java

```diff
@@ -287,6 +302,13 @@ protected static String getSchemaDiscoveryQuery(SourceSchemaReference sourceSche
         + " TABLE_NAME = ?";
   }
 
+  /**
+   * Discover Indexed columns and their Collations(if applicable). You could try this on <a href =
+   * https://https://www.db-fiddle.com/f/kRVPA5jDwZYNj2rsdtif4K/2>db-fiddle</a>
```

Comment:
this link seems broken

Reply:
Thanks. There was an extra https.
it's [here](https://www.db-fiddle.com/f/kRVPA5jDwZYNj2rsdtif4K/2)
Fixed in the current push.